phoenix-core-client/src/main/java/org/apache/phoenix/expression/function/DecodeFunction.java

```diff
@@ -59,7 +59,7 @@ public boolean evaluate(Tuple tuple, ImmutableBytesWritable ptr) {
 			return true; // expression was evaluated, but evaluated to null
 		}
 
-		String stringToDecode = (String) PVarchar.INSTANCE.toObject(ptr);
+        String stringToDecode = (String) PVarchar.INSTANCE.toObject(ptr);
```

Comment:
The indentation seems to be off.

Make sure to configure your IDE with the Phoenix standard code formatting setting
(then again, maybe this file is just misformatted.

In any case, try to minimize the whitespace changes, we're going to reformat the whole code base soon.

Reply:
I tried to follow that approach, however, the static code analyzer flagged some formatting issues, which I addressed to avoid any errors in the process.